common/levy_templates/01_celtic.txt

```diff
@@ -48,15 +48,16 @@ levy_norican = {
 levy_britannic = {
 	default = no
 
-	light_infantry = 0.6
+	light_infantry = 0.3
```

Comment:
this has 4 unit as well? 4 is less competitive because anything not listed in primary/secondary/flank comes into a battle with less morale

Reply:
Whenever there are more cohorts than combat length that will happen, it is not due to how many different types of units you have in the template, isn't it